Specialization/Protection.lua

```diff
@@ -48,64 +49,114 @@ local CN = {
 setmetatable(PR, Paladin.spellMeta);
 
 function Paladin:Protection()
+	MaxDps:GlowEssences();
+	Paladin:ProtectionCooldowns();
+
+	local fd = MaxDps.FrameData;
+	fd.targets = MaxDps:SmartAoe();
+	local targets = fd.targets;
+
+	if targets < 2 then
```

Comment:
from Icy: "Against two targets, our rotation stays the same as in single target, but against 3+ targets..."

I think you want targets < 3 here.

Reply:
Thx, so this is different to Retribution then. I'll change it